test/pobox_give_away_SUITE.erl

```diff
@@ -0,0 +1,134 @@
+-module(pobox_give_away_SUITE).
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-compile(export_all).
+
+all() -> [
+  owner_gives_away_ownership,
+  owner_gives_away_ownership_with_data,
+  owner_gives_away_ownership_to_dead_process,
+  foreign_process_tries_to_gives_away_ownership,
+  give_away_does_not_change_heir,
+  owner_gives_away_ownership_when_in_notify,
+  owner_gives_away_ownership_when_in_active
+].
+
+owner_gives_away_ownership(_Config) ->
+    {ok, Box} = pobox:start_link(#{size => 10}),
+    Self = self(),
+    H = erlang:spawn(fun F () -> receive X -> Self ! X end, F() end),
```

Comment:
hm, might want to spawn_link this one (and the others like it) so they at least get cleaned up in case of a test failure. I guess it wont matter on text success though since `start_link` of the PO Box will allow the cleanup from test -> pobox -> tmp proc to propagate

Reply:
I changed it where applicable